app/src/main/java/de/westnordost/streetcomplete/util/SoundFx.kt

```diff
@@ -9,13 +9,11 @@ import androidx.annotation.RawRes
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.suspendCancellableCoroutine
 import kotlinx.coroutines.withContext
-import javax.inject.Inject
-import javax.inject.Singleton
 import kotlin.coroutines.Continuation
 import kotlin.coroutines.resume
 
 /** Simple wrapper to enable just playing a sound effect from raw resources */
-@Singleton class SoundFx @Inject constructor(private val context: Context) {
+class SoundFx constructor(private val context: Context) {
```

Comment:
FYI you can just write

```suggestion
class SoundFx(private val context: Context) {
```